phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -667,6 +676,7 @@ public int run(String[] args) throws Exception {
         }
         populateIndexToolAttributes(cmdLine);
         Configuration configuration = getConfiguration(tenantId);
+        decorateConfWithProperties(configuration, cmdLine);
```

Comment:
Instead of calling this function here, why not do it inside the 678 to follow the setup code in the tool? You can extract properties in the function call at line no. 677 and use/pass in at 678.

Reply:
Updated the patch to do this.